src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/EmptyStatement/context/defaultContext.patch

```diff
@@ -7,6 +7,6 @@ index 7fe1740..8c68b53 100644
          {
              case 1 :
 -                break
-                 ;
+                 ;  // violation without filter
              default :
-                 ;
+                 ;  // violation without filter
```

Comment:
@timurt this is empty

Reply:
No, I mean do we have test for the following case?

- No violations before patching
- EmptyStatement violation after patching